### DISCUSSION TITLE
Absolute Beginner

### DISCUSSION THREAD

**Discussion by PeterLinu** [ampache/ampache#3369] [General]
Hiyall,
I produced and presented a radio show for 38 years until recently I was fired by the manager for being too old!
I am currently being helped setting up Ampache by a propellor-head.friend. 
My intention is to produce and create the same radio show as before as many ex-listeners miss the music. 
Questions;
1. Is it possible for each show (120 minutes) to be placed inside my web page with a date, eg: 'FJD221019'?
2. Initially, I don't expect more than 50 listeners. Would they all be able to click on FJD221019 and listen at the same time?
3. I want to prevent sharing by issuing a complex password. Is this facility available?

**Comment by lachlan-00** [comment]
That's interesting you want public links to media with a password/key? Yes.

You want a page where people can browse media about a single item (your show) sort of.

If you already have a site. The media can be easily added as links.

You've got an Interesting use case because shared links to media with a key/password are easily possible bit it sounds like you need more.

**Reply by PeterLinu** [reply]
No. That's all I need.
If I have a site, do I need Ampache?

**Reply by PeterLinu** [reply]
Sorry for the break in transmission...
That's interesting you want public links to media with a password/key? **Yes.**

You want a page where people can browse media about a single item (your show) sort of. **Not quite. I'd like each show to stay there for say, 6 months. That's ~24 shows.**

If you already have a site. The media can be easily added as links. **I have limited resources at home. I would prefer to use the Ampache cloud service.**

You've got an Interesting use case because shared links to media with a key/password are easily possible bit it sounds like you need more. **Username and password entry is essential.**
